src/k8s/fa_configmaps.tf

```
@@ -21,3 +21,17 @@ resource "kubernetes_config_map" "fa-jvm" {
     TZ = "Europe/Rome"
   }
 }
+
+
+resource "kubernetes_config_map" "fa-ms-customer" {
```

```suggestion
resource "kubernetes_config_map" "famscustomer" {
```
only for take a standard naming convention

---

yes, I agree. thank you